packages/frontend/src/map-contents/planted-dashboard/recently-planted-card/DataProcessor.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography } from '@mui/material';
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent';
+import { PUBLIC_VIEW_URL } from '../../../constants/ArcGIS';
+import { useEffect } from 'react';
+import { getAttachments, queryFeatures } from '@esri/arcgis-rest-feature-layer';
+import './RecentlyPlantedCard.scss'
+
+const CARDS_DISPLAYED : number = 5;
```

Comment:
nit: No space usually between variable name and the colon.

Also, this variable doesn't need a type. TS will implicitly know the type based on that assignment.